src/style.css

```diff
@@ -0,0 +1,118 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-items: center;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
create variable with colors that you use more than 1 time and use variable instead